mlflow/models/signature.py

```diff
@@ -90,6 +96,7 @@ def __init__(
         else:
             self.outputs = outputs
         self.params = params
+        self._is_signature_from_type_hint = False
```

Comment:
Can we make it a public property or define "is_signature_from_type_hint" property to avoid manipulating private property from outside of this class?

Reply:
Make sense, I'll add an internal property